### PR TITLE
LPS-199340 Add categories and tags in JournalManagementToolbarDisplayContext

### PR DESCRIPTION
**Pull request by veroglez**
Dear reviewer,

This code only includes the Tag and Category filters in the Management Toolbar, the filtering of the Web Contents will be done in this other task https://liferay.atlassian.net/browse/LPS-199515

https://github.com/liferay-echo/liferay-portal/assets/9701095/90c28bc5-3ba7-45ef-b409-a625c4e66e3a

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/JournalPortlet.java [hunk 1]
```diff
@@ -142,6 +143,9 @@ public void render(
 			AssetDisplayPageFriendlyURLProvider.class.getName(),
 			_assetDisplayPageFriendlyURLProvider);
 		renderRequest.setAttribute(TrashWebKeys.TRASH_HELPER, _trashHelper);
+		renderRequest.setAttribute(
```
Comment (jkappler) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/JournalPortlet.java:146]: @veroglez, we can sort attributes here

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java [hunk 2]
```diff
@@ -379,7 +412,13 @@ public List<LabelItem> getFilterLabelItems() {
 					LanguageUtil.get(httpServletRequest, "status") + ": " +
 						_getStatusLabel(status));
 			}
-		).build();
+		);
+
+		_addAssetTagsFilterLabelItems(labelItemListWrapper);
```
Comment (jkappler) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java:417]: @veroglez, we can also sort here

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java [hunk 3]
```diff
@@ -86,6 +101,13 @@ public JournalManagementToolbarDisplayContext(
 		_journalDisplayContext = journalDisplayContext;
 		_trashHelper = trashHelper;
 
+		_itemSelector = (ItemSelector)httpServletRequest.getAttribute(
```
Comment (jkappler) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalManagementToolbarDisplayContext.java:104]: @veroglez, also here

**Comment by ealonso** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:test:relevant

**Comment by veroglez** [comment]
see https://github.com/liferay-frontend/liferay-portal/pull/3776

**Pull request closed without merging**